### PR TITLE
quick fix on from_gridspec in structuredgrid class

### PR DESCRIPTION
**Pull request by briochh**
Was running into error setting coord info on cls. Think it should be on the instantiated grd object instead?

### CODE REVIEW

File: flopy/discretization/structuredgrid.py [hunk 1]
```diff
@@ -988,8 +988,8 @@ def from_gridspec(cls, gridspec_file, lenuni=0):
         grd = cls(np.array(delc), np.array(delr), lenuni=lenuni)
         xll = grd._xul_to_xll(xul)
         yll = grd._yul_to_yll(yul)
-        cls.set_coord_info(xoff=xll, yoff=yll, angrot=rot)
-        return cls
+        grd.set_coord_info(xoff=xll, yoff=yll, angrot=rot)
+        return grd
```
Comment (briochh) [inline flopy/discretization/structuredgrid.py:992]: is it ok to return a reconstructed grid object (i.e. not modifying modelgrid inplace)?

**Comment by mwtoews** [comment]
I also see that `from_gridspec` is absent from the autotest directory. There are also no `*.spc` files in this repo (but there are several in pyemu `git ls-files '*.spc'`). Obviously it would be nice to have at least one test for coverage.

File: flopy/discretization/structuredgrid.py (hunk 1 above)
Comment (briochh) [reply flopy/discretization/structuredgrid.py:992]: Modified as per @jlarsen-usgs suggestion. Good guess @langevin-usgs, not too desperate to dive down that rabbit hole! I am away from the computer for a short while now but can take a look at a test when I get back. Is there still appetite to continue to support gridspec files going forward?

**Comment by jdhughes-usgs** [comment]
@jlarsen-usgs @langevin-usgs if the latest CI passes are we good with merging this PR? I can add a issue for adding a test and assign it to @briochh

**Comment by jlarsen-usgs** [comment]
@jdhughes-usgs, I'm okay with it being merged in.

**Pull request merged**